### PR TITLE
replaced core map library

### PR DESCRIPTION
**Pull request by mohitb35**
replaced core map package(mapbox-gl -> maplibre-gl). 

Duplicate of #1999 which was deleted.

### CODE REVIEW

File: src/features/projects/components/maps/TimeTravel.tsx [hunk 1]
```diff
@@ -68,14 +67,14 @@ export default function TimeTravel(): ReactElement {
       // A selector or reference to HTML element
       const container = '#comparison-container';
 
-      const _compare = new MapboxCompare(before, after, container, {
-        mousemove: false, // Optional. Set to true to enable swiping during cursor movement.
+      const compare = new MapboxCompare(before, after, container, {
```
Comment (prachigarg19) [inline src/features/projects/components/maps/TimeTravel.tsx:70]: `compare` variable is not being used anywhere. Could you just declare the object without initializing the value to a variable.

File: src/features/projects/components/maps/TimeTravel.tsx
Comment (prachigarg19) [inline src/features/projects/components/maps/TimeTravel.tsx:1]: In this file `//@ts-ignore` has been used. This practice is not recommended. Is there a reason why this has been done? There aren't many errors in this file if we remove this.

**Review by prachigarg19 — CHANGES_REQUESTED** [review]

**Review by prachigarg19 — APPROVED** [review]

**Comment by norbertschuler** [comment]
Will the replacement of mapbox-gl with maplibre-gl be continued somewhere? Would be nice as it could reduce the overhead of code libraries to download a little bit for the website.

**Comment by mohitb35** [comment]
We are replacing mapbox-gl with maplibre-gl gradually. The new profile already uses this, and the upcoming facelift for the home page / project details page also uses maplibre.

**Comment by mohitb35** [comment]
Closing as not needed

**Pull request closed without merging**